content/ai/concepts/neural-networks/terms/binary-step-activation-function/binary-step-activation-function.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: 'Binary Step Activation Function'
+Description: 'A binary-step function is the simplest threshold based activation function which works by either firing the node in case the threshold value is surpassed or doing nothing at all.'
```

Comment:
```suggestion
Description: 'The simplest threshold based activation function which works by either firing the node in case the threshold value is surpassed or doing nothing at all.'
```
Changed due to how Docs displays terms: Title : Description. Because of this, mentioning the title in a term's description is redundant.